editor/components/block-list/style.scss

```diff
@@ -280,36 +326,40 @@
 		}
 	}
 
-	// Left and right side UI
-	> .editor-block-settings-menu,
-	> .editor-block-mover {
+
+	/**
+	 * 	Left and right side UI
```

Comment:
Nit: Tab here after comment asterisk should be removed.

Reply:
👍 👍